tests/tools.vitruv.change.atomic.tests/src/tools/vitruv/change/atomic/compound/RemoveAndDeleteRootTest.xtend

```diff
@@ -60,7 +55,7 @@ class RemoveAndDeleteRootTest extends EChangeTest {
 	@Test
 	def void resolveToCorrectType() {
 		// Create change
-		val unresolvedChange = createUnresolvedChange(newRootObject, 0)
+		val unresolvedChange = createUnresolvedChange(newRootObject, 1)
```

Comment:
I could not trace the reason for this change (also in `applyBackwardTest`).
Same in `RemoveRootEObjectTest`.

Reply:
Comment from @JanWittler "This was just always wrong, just didn't notice because of HIDs. If you look at the tests in detail, the index must be like the new value from the semantics, otherwise index and element are not consistent."

Reply:
👍